src/main/java/seedu/address/logic/commands/find/FindProfileCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands.find;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.util.CommandUtil.getCommandResult;
+import static seedu.address.model.util.ItemUtil.PROFILE_ALIAS;
+import static seedu.address.model.util.ItemUtil.PROFILE_NAME;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.profile.ProfileItemContainsKeywordPredicate;
+import seedu.address.ui.tabs.TabName;
+
+/**
+ * Finds and lists all profileItem in Internhunter whose title contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindProfileCommand extends FindCommand {
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " "
```

Comment:
Should we stick to how other messages are done, I realised we are missing `the Parameters: ...` 
Might break some test cases message.